lib/active_admin/localizers/resource_localizer.rb

```diff
@@ -0,0 +1,35 @@
+module ActiveAdmin
+  module Localizers
+    class ResourceLocalizer
+      class << self
+        def from_resource(resource_config)
+          new(resource_config.resource_name.i18n_key, resource_config.resource_label)
+        end
+
+        def translate(key, options)
+          new(options.delete(:model_name), options.delete(:model)).translate(key, options)
+        end
+        alias_method :t, :translate
+      end
+
+      def initialize(model_name, model = nil)
+        @model_name = model_name
+        @model = model || model_name.to_s.titleize
+      end
+
+      def translate(key, options = {})
+        scope = options.delete(:scope)
+        specific_key = array_to_key('resources', @model_name, scope, key)
+        defaults = [array_to_key(scope, key), key.to_s.titleize]
+        ::I18n.t specific_key, options.reverse_merge(model: @model, default: defaults, scope: 'active_admin')
```

Comment:
Line is too long. [109/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.